jobs/azure-discovery/src/main/java/com/tmobile/pacbot/azure/inventory/collector/SecurityAlertsInventoryCollector.java

```diff
@@ -48,7 +48,9 @@ public List<SecurityAlertsVH> fetchSecurityAlertsDetails(SubscriptionVH subscrip
 				securityAlertsVH.setType(databricksObject.get("type").getAsString());
 				securityAlertsVH.setSubscription(subscription.getSubscriptionId());
 				securityAlertsVH.setSubscriptionName(subscription.getSubscriptionName());
-				securityAlertsVH.setRegion(Util.getRegionValue(subscription,databricksObject.get("location").getAsString()));
+				String id=securityAlertsVH.getId();
```

Comment:
I'd like to have this line up top, around line 46, so you can use this value for `setId` method and for region as well.
And it should look like this
```
String id = databricksObject.get("id").getAsString();
```

Reply:
Resolved in commit